src/helpers/moderation.ts

```diff
@@ -1,6 +1,7 @@
 import snapshot from '@snapshot-labs/snapshot.js';
-import log from './log';
 import { capture } from '@snapshot-labs/snapshot-sentry';
+import fetch from 'cross-fetch';
```

Comment:
Because ofetch is just a wrapper around node-fetch when used on node env.

Reply:
ok, but we decided to move from `cross-fetch` to `ofetch`, no?

Reply:
Only when `cross-fetch` are used for browser and node support.

This repo is using `cross-fetch` for no particular reason, since browser support is not needed. 

Using `ofetch` instead of `node-fetch` will not add much benefit, as under the hood, ofetch still use node-fetch.

Reply:
I'm getting confused. Still using `cross-fetch` in this PR, will be migrated to `node-fetch` only in this next PR (https://github.com/snapshot-labs/snapshot-sequencer/pull/179)